### ISSUE TITLE
[TESTING] test restore script v1.0.0

### ISSUE THREAD

**Issue by watchfulli** [watchfulli/XCloner-WordPress-restore#2]
# Target site details & prep
1. Site name: Site 001
1. DB Prefix: wp602a_
- [x] Download testing version of restore script.
- [x] Extract testing script.
- [x] Upload testing script (including the `vendor` folder) to root of target site.

# Source site details & prep
1. Site name: Site 002
- [x] Install most recent version of XCloner (or testing version as appropriate).
- [x] Create a full site backup.
  - [x] If database is used for multiple sites, be sure to only select the tables for the site you are backing up
  - Backup size: Enter size of backup in MB.
  
# Testing the restore process
- [x] Visit the restore area of XCloner on the source site. 
- [x] Enter the URL of the restore script and click `Check Connection`.
  - [x] Connection is good.
- [x] Select the backup created on the source site from the dropdown and click `Upload`.
  - [x] Upload completes.
- [x] Click `Next` to restore the files.
  - [x] File restoration completes.
- [x] Click `Next` to view the database details.
  - [x] Enter the database details.
- [x] Click `Next` to restore the database.
- [x] Confirmation & Cleanup
  - [x] Target site
    - [x] Target site loads cloned website.
    - [x] Upload script deleted from target site.
    - [x] Backup archive deleted from target site.
    - [x] Extracted archive folder deleted from target site.

**Comment by jimiero** [comment]
getting this error while trying to restore ( used site was : https://php74.watchful.pro/wp602a/wp-admin/ )

`/10/07 10:30:52 [error] 24535#24535: *2143143 FastCGI sent in stderr: "PHP message: PHP Fatal error:  Uncaught RuntimeException: SplFileInfo::getType(): Lstat failed for /home/demos/webapps/php74/public/wp602a/wp-content/backups-GgoVN//.xcloner/index.html in /home/demos/webapps/php74/public/wp602a/wp-content/plugins/xcloner-backup-and-restore/vendor/league/flysystem/src/Adapter/Local.php:509
#3 /home/demos/webapps/php74/public/wp602a/wp-content/plugins/xcloner-backup-and-restore/vendor/league/flysystem/src/Filesystem.php" while reading response header from upstream, client: 172.70.242.7, server: php74.watchful.pro, request: "POST /wp602a/wp-admin/admin-ajax.php?_wpnonce=219af9feec HTTP/2.0", upstream: "fastcgi://unix:/var/run/php74.sock:", host: "php74.watchful.pro", referrer: "https://php74.watchful.pro/wp602a/wp-admin/admin.php?page=xcloner_generate_backups_page"
#2 /home/demos/webapps/php74/public/wp602a/wp-content/plugins/xcloner-backup-and-restore/vendor/league/flysystem/src/Adapter/Local.php(305): League\Flysystem\Adapter\Local->normalizeFileInfo()
#1 /home/demos/webapps/php74/public/wp602a/wp-content/plugins/xcloner-backup-and-restore/vendor/league/flysystem/src/Adapter/Local.php(452): League\Flysystem\Adapter\Local->mapFileInfo()
#0 /home/demos/webapps/php74/public/wp602a/wp-content/plugins/xcloner-backup-and-restore/vendor/league/flysystem/src/Adapter/Local.php(509): SplFileInfo->getType()
Stack trace:






`

**Comment by vdrover** [comment]
When do you get this error exactly? The issue indicates you only uploaded the script and stopped. Please update with more details.

**Comment by jimiero** [comment]
I get the error when running a full website backup in XCloner to use it for restore proces

**Comment by vdrover** [comment]
@jimiero Please proceed with testing

**Comment by vdrover** [comment]
The restore worked for me.
